fetchy/fetchyboi.ts

```diff
@@ -2,40 +2,62 @@ import getVaccineData from "./vaccineDataService";
 import {
   createVaccineDataForDate,
   getVaccineDataForDate,
-  updateVaccineDataIfDifferentForDate,
+  updateVaccineDataForDate,
 } from "../shared/dataRepository";
 import got from "got";
+import { diff } from "deep-object-diff";
+
+const sendNotification = async (message: string) => {
+  // don't send notifications if running in a PR
+  if (process.env.GITHUB_EVENT_NAME === "pull_request") {
```

Comment:
Does this send notifications if you run `fetchyboi` locally? Maybe we should make this an opt in to messaging and then we set an env var in the github action?

```suggestion
  if (process.env.SEND_DISCORD_NOTIFICATIONS === "true") {
```

Reply:
Ooo yes much better idea. Will do in another pr